### PR TITLE
Add e2e test for aggregates info buttons

### PR DESCRIPTION
**Pull request by Piotrk39**
## Describe your changes

Adds test for validating Aggregates' component info buttons' contents.

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] I have added tests wherever relevant
- [x] I have made sure that all automated checks pass before requesting a review

## Notes for the reviewer

1. For this test to be as flexible as possible I have selected main info Icon (title) and Exposed population layer's icon since I believe they are always present regardless the disaster type.

### CODE REVIEW

File: tests/e2e/Pages/AggregatesComponet.ts [hunk 1]
```diff
@@ -27,14 +31,20 @@ class AggregatesComponent extends DashboardPage {
...
-    );
     this.aggregatesInfoIcon = this.page.getByTestId('aggregates-info-icon');
     this.aggregatesLayerRow = this.page.getByTestId('aggregates-row');
     this.aggregatesAffectedNumber = this.page.getByTestId(
       'aggregates-affected-number',
     );
+    this.aggreagtesTitleInfoIcon = this.page.getByTestId(
+      'aggregates-title-info-icon',
+    );
+    this.approximatedisclaimer = this.page.getByTestId(
```
Comment (gulfaraz) [inline tests/e2e/Pages/AggregatesComponet.ts:42]: Use camelCase for variables.

File: tests/e2e/Pages/AggregatesComponet.ts [hunk 2]
```diff
@@ -73,6 +85,43 @@ class AggregatesComponent extends DashboardPage {
       await expect(layerLocator).toBeVisible();
     }
   }
+
+  async validatesAggregatesInfoButtons() {
+    // click on the first info icon and validate the opopver content
+    await this.aggreagtesTitleInfoIcon.click();
+    const disclaimerText = await this.approximatedisclaimer.textContent();
+    expect(disclaimerText).toContain(
+      'All numbers are approximate and meant to be used as guidance.',
```
Comment (gulfaraz) [inline tests/e2e/Pages/AggregatesComponet.ts:94]: I'd move these strings to a helper file. Design may decide to change these strings. I'd prefer if these strings are separated from the test logic.

File: tests/e2e/Pages/AggregatesComponet.ts [hunk 3]
```diff
@@ -73,6 +85,43 @@ class AggregatesComponent extends DashboardPage {
...
+    expect(layerInfoTitle).toContain('Exposed population');
+    expect(layerInfoContent).toContain(
+      'This layer shows the estimated rounded number of people potentially exposed per geographic area.',
+    );
+  }
+
+  async validateLayerPopoverExternalLink() {
+    // Define link to click
+    const layerPopoverExternalLink = this.layerInfoPopoverContent.filter({
+      hasText: 'High Resolution Settlement Layer (HRSL)',
```
Comment (gulfaraz) [inline tests/e2e/Pages/AggregatesComponet.ts:117]: I believe the sources may be different depending on the country-hazard. We'll deal with the differences for other country-hazard later.

Move the strings to a separate helper file so dealing with the differences in string values is separated from the test logic.

**Review by gulfaraz — CHANGES_REQUESTED** [review]

File: tests/e2e/Pages/AggregatesComponet.ts (hunk 2 above)
Comment (Piotrk39) [reply tests/e2e/Pages/AggregatesComponet.ts]: Fixed this and another one in different PR: https://github.com/rodekruis/IBF-system/pull/1658

File: tests/e2e/Pages/AggregatesComponet.ts (hunk 3 above)
Comment (Piotrk39) [reply tests/e2e/Pages/AggregatesComponet.ts:118]: Got It! Fixed by mistake in different PR: https://github.com/rodekruis/IBF-system/pull/1658

File: tests/e2e/Pages/AggregatesComponet.ts (hunk 1 above)
Comment (Piotrk39) [reply tests/e2e/Pages/AggregatesComponet.ts]: Done

File: tests/e2e/Pages/AggregatesComponet.ts (hunk 3 above)
Comment (gulfaraz) [reply tests/e2e/Pages/AggregatesComponet.ts:118]: > Got It! Fixed by mistake in different PR: https://github.com/rodekruis/IBF-system/pull/1658

@Piotrk39 Does this mean there's nothing left to review in this PR? If the review comments are resolved else and there's nothing left to review here. Please feel free to merge.

Comment (Piotrk39) [reply tests/e2e/Pages/AggregatesComponet.ts:118]: Yes indeed. I'll merge

**Pull request merged**